span.go

```diff
@@ -222,6 +226,25 @@ func (r *spanS) sendOpenTracingLogRecord(lr ot.LogRecord) {
 	)
 }
 
-func isErrorLogField(lf otlog.Field) bool {
-	return lf.Key() == "error" || lf.Key() == "error.object"
+func openTracingLogRecordLevel(lr ot.LogRecord) logger.Level {
```

Comment:
What do you think if we will rename this function to something like `highestLogRecordLevel` or similar?

Reply:
Good point, that would be more descriptive method name 👍